client/src/components/browseRecipes/Browse.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import { onMount } from 'svelte';
+  import Header from '@/components/layout/Header.svelte';
+  import Navigation from '@/components/layout/Navigation.svelte';
+  import Category from '@/components/browseRecipes/Category.svelte';
+  import Recipes from '@/components/browseRecipes/Recipes.svelte';
+  import { recipesApi } from '@/api';
+
+  let category = 0;
+  let recipes = [];
+
+  function handleCategoryToggle({ detail: { value } }) {
+    category = +value;
+  }
+
+  onMount(async () => {
+    recipes = await recipesApi.fetchAll();
+  });
+</script>
+
+<main>
+  <Header />
+  <Navigation />
+  <Category on:categoryToggle={handleCategoryToggle} />
```

Comment:
once we rename this to `on:update` we can rename the function to `updateCategory` since that would make more sense then toggle 😄